libc.in

```diff
@@ -226,8 +228,14 @@ if $backup; then
 	fi
 fi
 
+# Protect against symlink attack, ensure new file does not exist
+rm -f "$resolv_conf_tmp" 2>/dev/null || :
+# Keep original file owner, group and mode
+[ -r "$resolv_conf" ] && cp -p "$resolv_conf" "$resolv_conf_tmp" || :
```

Comment:
We want cp to fail noisly I think, so remove the trailing `|| :` Maybe like this?
`[ ! -r "$resolv_conf" ] || cp -p "$resolv_conf" "$resolv_conf_tmp"`

Or if that is confusing use `if [ -r "$resolv_conf" ]; then`

Reply:
I just removed || : suffix, but left && intact. It were used to continue even in case of failure, but openresolv does not use set -e to abort sooner. It is not needed and so removed.